### PR TITLE
parse timestamps for changelog

### PR DESCRIPTION
**Pull request by Rots**

### CODE REVIEW

File: issue.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ type ChangelogItems struct {
 type ChangelogHistory struct {
 	Id      string           `json:"id" structs:"id"`
 	Author  User             `json:"author" structs:"author"`
-	Created string           `json:"created" structs:"created"`
+	Created Time           `json:"created" structs:"created"`
```
Comment (rbriski) [inline issue.go]: This will break all code that is expecting a string.  Also, there are no tests to check that this actually works in different situations (bad format, empty string, positive case, etc).

If you want to add a field or possibly just add a function that returns the field as a `Time`.  Something like `CreatedAsTime()`, that would work.

**Comment by rbriski** [comment]
@Rots Any updates here?  Otherwise, I'm going to close this PR.

**Comment by Rots** [comment]
I will add the suggested method to a few places where timestamp is used

**Comment by Rots** [comment]
Not sure how to handle the existing `func (c ChangelogHistory) CreatedTime() (time.Time, error)` notice the return type is `time.Time` instead of `Time`.

**Comment by ghostsquad** [comment]
I agree that adding a helper method here would allow for the desired effect while keeping backwards compatibility.

**Comment by ghostsquad** [comment]
This PR is getting stale. I'll be closing in next week if no one is looking at it.

**Comment by andygrunwald** [comment]
hey @Rots,

thanks for taking the time for this PR.
Any updates on thisß
